apps/web/src/pages/api/forms/organisation.ts

```diff
@@ -15,7 +15,7 @@ import type { OrganisationAddInputs } from '@/utils/schemas'
 import { organisationAddSchema } from '@/utils/schemas'
 import { withApiHandler } from '@/utils/withApiHandler'
 
-import { assignGroupToUser, isUserEligibleForOdpRole } from './registration'
+import { addUserGroup, isUserEligibleForOdpRole } from './registration'
```

Comment:
I think this naming is potentially confusing.
Is it adding a 'User Group' or adding a User to a Group?

Suggest renaming this to `addUserToGroup`

Reply:
updated